### PR TITLE
[IT-1636] EC2 instance for gene co-expression network

### PR DESCRIPTION
**Pull request by zaro0508**
requested by @jgockley62 

depends on https://github.com/Sage-Bionetworks/aws-infra/pull/334

### CODE REVIEW

File: config/prod/EC2-gene-co-expression.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.38/EC2/jc-ec2-linux.j2"
+stack_name: "EC2-gene-co-expression"
+stack_tags:
+  Department: "TBD"
+  Project: "TBD"
+  OwnerEmail: "jake.gockley@sagebase.org"
+  CostCenter: "TBD"
```
Comment (zaro0508) [inline config/prod/EC2-gene-co-expression.yaml:9]: @jgockley62 please advise on values that should be set in place of `TBD`

cost centers reference: https://github.com/Sage-Bionetworks/aws-infra/tree/master/templates/tags

File: config/prod/EC2-gene-co-expression.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+sceptre_user_data:
+  Distro: "aws"
+parameters:
+  JcUserId: "jake.gockley@sagebase.org"
+  InstanceType: "c6i.32xlarge"
+  VolumeSize: "100"        # in GB
```
Comment (zaro0508) [inline config/prod/EC2-gene-co-expression.yaml:15]: @jgockley62 please verify these values.

Comment (jgockley62) [reply config/prod/EC2-gene-co-expression.yaml:15]: @zaro0508 Can we do 200GB just in case?

File: config/prod/EC2-gene-co-expression.yaml (hunk 1 above)
Comment (jgockley62) [reply config/prod/EC2-gene-co-expression.yaml:9]: @zaro0508 
```
  Department: "ADTR"
  Project: "ConsensusNetworks"
  OwnerEmail: "jake.gockley@sagebase.org"
  CostCenter: "NIA AMP-AD CC / 101500"
```

**Pull request merged**